<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>dlg_swapaxis</class>
 <widget class="QDialog" name="dlg_swapaxis">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>378</width>
    <height>244</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Swap Axis</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QGroupBox" name="groupBox">
     <property name="title">
      <string>New X Axis</string>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QLabel" name="textLabel1">
        <property name="text">
         <string>Old Axis:</string>
        </property>
        <property name="wordWrap">
         <bool>false</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="m_pNewX_Flip">
        <property name="text">
         <string>-</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="m_pNewX_X">
        <property name="text">
         <string>X</string>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
        <attribute name="buttonGroup">
         <string>m_pButtonGroupX</string>
        </attribute>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="m_pNewX_Y">
        <property name="text">
         <string>Y</string>
        </property>
        <attribute name="buttonGroup">
         <string>m_pButtonGroupX</string>
        </attribute>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="m_pNewX_Z">
        <property name="text">
         <string>Z</string>
        </property>
        <attribute name="buttonGroup">
         <string>m_pButtonGroupX</string>
        </attribute>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBox_2">
     <property name="title">
      <string>New Y Axis</string>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <item>
       <widget class="QLabel" name="textLabel1_2">
        <property name="text">
         <string>Old Axis:</string>
        </property>
        <property name="wordWrap">
         <bool>false</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="m_pNewY_Flip">
        <property name="text">
         <string>-</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="m_pNewY_X">
        <property name="text">
         <string>X</string>
        </property>
        <attribute name="buttonGroup">
         <string>m_pButtonGroupY</string>
        </attribute>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="m_pNewY_Y">
        <property name="text">
         <string>Y</string>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
        <attribute name="buttonGroup">
         <string>m_pButtonGroupY</string>
        </attribute>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="m_pNewY_Z">
        <property name="text">
         <string>Z</string>
        </property>
        <attribute name="buttonGroup">
         <string>m_pButtonGroupY</string>
        </attribute>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBox_3">
     <property name="title">
      <string>New Z Axis</string>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_3">
      <item>
       <widget class="QLabel" name="textLabel1_3">
        <property name="text">
         <string>Old Axis:</string>
        </property>
        <property name="wordWrap">
         <bool>false</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="m_pNewZ_Flip">
        <property name="text">
         <string>-</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="m_pNewZ_X">
        <property name="text">
         <string>X</string>
        </property>
        <attribute name="buttonGroup">
         <string>m_pButtonGroupZ</string>
        </attribute>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="m_pNewZ_Y">
        <property name="text">
         <string>Y</string>
        </property>
        <attribute name="buttonGroup">
         <string>m_pButtonGroupZ</string>
        </attribute>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="m_pNewZ_Z">
        <property name="text">
         <string>Z</string>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
        <attribute name="buttonGroup">
         <string>m_pButtonGroupZ</string>
        </attribute>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout">
     <item>
      <spacer name="spacer1">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeType">
        <enum>QSizePolicy::Expanding</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>50</width>
         <height>30</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <layout class="QHBoxLayout">
       <item>
        <widget class="QPushButton" name="m_pOKButton">
         <property name="text">
          <string>OK</string>
         </property>
         <property name="default">
          <bool>false</bool>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="pushButton2">
         <property name="text">
          <string>Cancel</string>
         </property>
         <property name="default">
          <bool>true</bool>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <spacer name="spacer2">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeType">
        <enum>QSizePolicy::Expanding</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>50</width>
         <height>30</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <pixmapfunction>qPixmapFromMimeSource</pixmapfunction>
 <resources/>
 <connections>
  <connection>
   <sender>pushButton2</sender>
   <signal>clicked()</signal>
   <receiver>dlg_swapaxis</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>20</x>
     <y>20</y>
    </hint>
    <hint type="destinationlabel">
     <x>20</x>
     <y>20</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>m_pOKButton</sender>
   <signal>clicked()</signal>
   <receiver>dlg_swapaxis</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>20</x>
     <y>20</y>
    </hint>
    <hint type="destinationlabel">
     <x>20</x>
     <y>20</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <buttongroups>
  <buttongroup name="m_pButtonGroupX"/>
  <buttongroup name="m_pButtonGroupY"/>
  <buttongroup name="m_pButtonGroupZ"/>
 </buttongroups>
</ui>
